db/migrate/20190912121620_create_conversations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateConversations < ActiveRecord::Migration[5.2]
+  def change
+    create_table(:conversations, &:timestamps)
```

Comment:
Why this?

Reply:
I don't know what is this doing here. This is from old commits, probably messed it when rebasing. Will rebase again before upload to see if it goes away.